homeassistant/components/config/config_entries.py

```diff
@@ -334,14 +338,18 @@ async def config_entry_update(hass, connection, msg):
     }
 )
 @websocket_api.async_response
-async def config_entry_disable(hass, connection, msg):
+async def config_entry_disable(
+    hass: HomeAssistant,
+    connection: websocket_api.ActiveConnection,
+    msg: dict[str, Any],
+) -> None:
     """Disable config entry."""
     if (disabled_by := msg["disabled_by"]) is not None:
         disabled_by = config_entries.ConfigEntryDisabler(disabled_by)
 
-    result = False
+    success = False
```

Comment:
I renamed the variable here because `mypy` didn't like the re-redefining on line 354 (= new line 362)